templates/patterns/inpage_navigation/inpage_navigation.ui_patterns.yml

```diff
@@ -0,0 +1,37 @@
eu deserunt sunt. Mollit qui sunt commodo id voluptate voluptate Lorem dolore est voluptate mollit consectetur ullamco est. Enim minim sunt ex non pariatur ex labore qui laborum sint consequat. Aute cupidatat id sint incididunt Lorem Lorem aute aliqua sint aliquip commodo laborum quis. Ex irure sit reprehenderit in amet. Fugiat in amet proident consequat occaecat officia et consequat ipsum aute commodo. Pariatur duis consequat ipsum enim quis Lorem sint eiusmod esse. Occaecat amet esse aliquip non incididunt nisi sint do. Magna deserunt irure est nostrud deserunt in elit in enim do occaecat consequat. Incididunt duis reprehenderit commodo mollit culpa ex. Cillum ullamco cupidatat sit fugiat exercitation voluptate elit nulla duis. Dolor deserunt ad aute sint. Sunt cillum aute anim quis anim anim. Voluptate enim proident voluptate deserunt mollit. Veniam exercitation ex aliquip laborum commodo esse laboris. Veniam exercitation dolore quis ea minim magna. Labore adipisicing magna in excepteur esse ullamco aliqua tempor veniam id anim. Nulla deserunt laboris aute reprehenderit nisi quis commodo. Excepteur tempor nulla amet fugiat ullamco sit. Nostrud proident exercitation veniam elit ullamco ullamco officia minim officia commodo id duis. Laboris ullamco aliquip voluptate dolore aliquip nostrud magna ut cillum nisi officia.</p><p>Et pariatur qui quis ullamco magna esse aliqua. Ea eiusmod commodo in laborum nulla cillum anim. Culpa sint dolor aliquip pariatur proident ipsum id. Est voluptate et eiusmod labore ea ad consectetur minim nostrud ad. Exercitation sint laborum velit consequat magna dolor voluptate dolore incididunt nostrud eu deserunt sunt. Mollit qui sunt commodo id voluptate voluptate Lorem dolore est voluptate mollit consectetur ullamco est. Enim minim sunt ex non pariatur ex labore qui laborum sint consequat. Aute cupidatat id sint incididunt Lorem Lorem aute aliqua sint aliquip commodo laborum quis. Ex irure sit reprehenderit in amet. Fugiat in amet proident consequat occaecat officia et consequat ipsum aute commodo. Pariatur duis consequat ipsum enim quis Lorem sint eiusmod esse. Occaecat amet esse aliquip non incididunt nisi sint do. Magna deserunt irure est nostrud deserunt in elit in enim do occaecat consequat. Incididunt duis reprehenderit commodo mollit culpa ex. Cillum ullamco cupidatat sit fugiat exercitation voluptate elit nulla duis. Dolor deserunt ad aute sint. Sunt cillum aute anim quis anim anim. Voluptate enim proident voluptate deserunt mollit. Veniam exercitation ex aliquip laborum commodo esse laboris. Veniam exercitation dolore quis ea minim magna. Labore adipisicing magna in excepteur esse ullamco aliqua tempor veniam id anim. Nulla deserunt laboris aute reprehenderit nisi quis commodo. Excepteur tempor nulla amet fugiat ullamco sit. Nostrud proident exercitation veniam elit ullamco ullamco officia minim officia commodo id duis. Laboris ullamco aliquip voluptate dolore aliquip nostrud magna ut cillum nisi officia.</p>"
```

Comment:
can we make the content a bit smaller, just seems a bit much

Reply:
In order to see the effect of inpage we need a longer text than in the rest of elements, which you can see at /patterns page or at the example of netlify. 
If you still need a shorter text, I'll put it, but consider that then the effect of scrollspy can be lost if text is small.